### DISCUSSION TITLE
How to disable grub2 menu edit text?

### DISCUSSION THREAD

**Discussion by steve6375** [a1ive/grub2-filemanager#322] [General]
Is there a way to prevent the 'Use the ^ and V keys...' menu help text being displayed?

I have set text mode and 
```
export grub_disable_console=1
export grub_disable_edit=1
```

![image](https://user-images.githubusercontent.com/1469350/103466017-ec209e00-4d38-11eb-9b93-61a1c486255e.png)

Also, is there a way to get more than 10 entries per screen?

**Comment by a1ive** [comment]
> Also, is there a way to get more than 10 entries per screen?

![深度截图_选择区域_20210103120855](https://user-images.githubusercontent.com/10670106/103471521-84139b80-4dbc-11eb-8219-7f60148b0748.png)

**Comment by a1ive** [comment]
> Is there a way to prevent the 'Use the ^ and V keys...' menu help text being displayed?

If the text here is not displayed, then the text of '--help-msg' will not be displayed either.
The number of menu entries and the position of the countdown message are more difficult to calculate.

**Reply by steve6375** [reply]
Sorry! I can't count! more than 12 entries. If difficult then don't worry.

The help message is misleading though because e and c and b dont work if disabled, and ESC does not return to 'previous' menu.

re, help message - Could you patch beginning of the message to 0 so it is not displayed? grub_usage_msg_disable=1 ?
